### PR TITLE
Adds the createModel.spec.js

### PR DESCRIPTION
**Pull request by mAiNiNfEcTiOn**
# What does this PR do:
  - Adds the createModel.spec.js.

screen shot 2016-07-13 at 09 28 53

# Where should the reviewer start:
  - Diffs
  - `npm test`

# Unit and/or functional tests:
Added the test `createModel.spec.js`

### CODE REVIEW

File: test/createModel.spec.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* global describe, it */
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
```
Comment (fahad19) [inline test/createModel.spec.js]: we need to load sinon in this spec?

File: test/createModel.spec.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import BaseModel from '../src/Model';
+import createModel from '../src/createModel';
+chai.use(sinonChai);
```
Comment (alexmiranda) [inline test/createModel.spec.js]: This is becoming repetitive here. Can we use `--require` on `mocha.opts` to load a helper or something? It doesn't need to be in this PR, just wanted to know your opinion and I can make the change in a separate PR along with other tests.

File: test/createModel.spec.js (hunk 1 above)
Comment (mAiNiNfEcTiOn) [inline test/createModel.spec.js]: Removing sir.

File: test/createModel.spec.js (hunk 2 above)
Comment (mAiNiNfEcTiOn) [inline test/createModel.spec.js]: Agree, but then again, you don't need to load it all the time :) This is a case ;-)

**Comment by alexmiranda** [comment]
LGTM

**Pull request merged**